### ISSUE TITLE
EMAGE: traning uses 'emage' model while inference uses 'emage_audio' model

### ISSUE THREAD

**Issue by weihaosky** [PantoMatrix/PantoMatrix#146]
Hi, 
In the training, the code trains an emage mode. But in the inference, the code needs an emage_audio model.
How to transfer emage model to emage_audio model?

**Comment by H-Liu1997** [comment]
hi 
emage_audio is for our hugging face demo. if you train with emage, you may run test_emage instead of test_emage_audio. 
thanks!

**Comment by weihaosky** [comment]
> hi emage_audio is for our hugging face demo. if you train with emage, you may run test_emage instead of test_emage_audio. thanks!

Yes I know. I am asking how to train an emage_audio model. Thanks!

**Comment by H-Liu1997** [comment]
@weihaosky hi, could you take a look of current codebase,

you could train emage audio with
`torchrun --nproc_per_node 1 --nnodes 1 train_emage_audio.py --config ./configs/emage_audio.yaml --evaluation` now.

Thanks

**Comment by weihaosky** [comment]
> @weihaosky hi, could you take a look of current codebase,
> 
> you could train emage audio with `torchrun --nproc_per_node 1 --nnodes 1 train_emage_audio.py --config ./configs/emage_audio.yaml --evaluation` now.
> 
> Thanks

Ok, thanks!
